scene/gui/control.cpp

```diff
@@ -1666,17 +1669,14 @@ void Control::_update_minimum_size_cache() {
 	minsize.x = MAX(minsize.x, data.custom_minimum_size.x);
 	minsize.y = MAX(minsize.y, data.custom_minimum_size.y);
 
-	bool size_changed = false;
-	if (data.minimum_size_cache != minsize) {
-		size_changed = true;
-	}
+	data.minimum_size_valid = true; // Reset to true anyway.
```

Comment:
Ideally, `update_minimum_size()` would only be called when the minimum size has changed, but in reality the minimum size may not have changed.